packages/web3-wrapper/src/web3_wrapper.ts

```diff
@@ -562,9 +562,10 @@ export class Web3Wrapper {
         }
         const marshalledDefaultBlock = marshaller.marshalBlockParam(defaultBlock);
         const callDataHex = marshaller.marshalCallData(callData);
+        const overrides = marshaller.marshalCallOverrides(callData.overrides || {});
         const rawCallResult = await this.sendRawPayloadAsync<string>({
             method: 'eth_call',
-            params: [callDataHex, marshalledDefaultBlock],
+            params: [callDataHex, marshalledDefaultBlock, overrides],
```

Comment:
will this cause any problems for non-geth clients?

Reply:
ok the CI failures tell me yes –– check if overrides is empty before adding it to params?

Reply:
doh